modules/nf-core/salmon/index/tests/main.nf.test

```diff
@@ -32,4 +32,28 @@ nextflow_process {
 
     }
 
+     test("sarscov2 stub") {
+        options "-stub"
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                """
+                input[0] = Channel.of([file(params.modules_testdata_base_path + "genomics/homo_sapiens/genome/genome.fasta", checkIfExists: true)])
+                input[1] = Channel.of([file(params.modules_testdata_base_path + "genomics/sarscov2/genome/transcriptome.fasta", checkIfExists: true)])
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert path(process.out.index.get(0)).exists() },
```

Comment:
```suggestion
                { assert path(process.out.results.get(0)).exists() },
```

`index` is not a valid output path, I guess you want to use the `results` channel instead?

Reply:
That one is for the salmon/index module, the test passes, the one that causes the issue is the nf-test for salmon/quant